bin/assembleCoadd.py

```diff
@@ -21,6 +21,28 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 import lsst.pex.logging as pexLog
-from lsst.pipe.tasks.assembleCoadd import AssembleCoaddTask
+from lsst.pipe.tasks.assembleCoadd import AssembleCoaddTask, SafeClipAssembleCoaddTask
+import sys
+import argparse
 
-AssembleCoaddTask.parseAndRun()
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--legacyCoadd', action='store_true', default=False)
+parser.add_argument('-h','--help', action='store_true',default=False)
+result, extra = parser.parse_known_args(sys.argv)
+
+if result.help:
+    print('Meta Arguments:\n  --legacyCoadd \t\tAn option to run with the original assembleCoadd '+\
```

Comment:
does this not use triple-quote because of the tabs? Also, I don't think the `+` is needed because python concatenates quoted strings automatically:
```python
a = "abc" "def"
print(a)
```
will print "abcdef". I also think a space is missing between the final two strings.